scripts/deploy/version_bump_pr_steps.rb

```diff
@@ -64,14 +61,6 @@ def create_version_bump_pr()
     wait_for_user
 end
 
-def revert_all_changes()
```

Comment:
I realized that we don't actually need to do this -- we can just delete the branch that all of this has been committed to